docs/packs/packs-format.md

```diff
@@ -66,6 +66,7 @@ The following fields are populated in the pack metadata:
 | `devEmail` | `List` | (Optional) List of emails to receive notification in case contributed pack files were modified. |
 | `certification` | `String` | (Optional) If the pack is certified the value of this fields should be `certified`. The allowed values are `certified` and `verified`. |
```

Comment:
minor update:
(Optional) If the pack is certified, the value of this field should be `certified`. Possible values are `certified` or `verified`.

Reply:
```suggestion
| `certification` | `String` | (Optional) If the pack is certified, the value of this field should be `certified`. Possible values are `certified` or `verified`. |
```